typings/index.d.ts

```diff
@@ -46,6 +46,12 @@ declare module '*.svg' {
   export default content;
 }
 
+declare module '*.txt' {
```

Comment:
curious, why add this?

Reply:
The ```curl_parsing.test``` imports txt file for the testing purposes and TS required module declaration